dvc/repo/__init__.py

```diff
@@ -85,6 +85,8 @@ def __init__(self, root_dir=None):
 
         self.tmp_dir = os.path.join(self.dvc_dir, "tmp")
         makedirs(self.tmp_dir, exist_ok=True)
+        self.index_dir = os.path.join(self.dvc_dir, "index")
+        makedirs(self.index_dir, exist_ok=True)
```

Comment:
Forgetting to gitignore it :) But actually, maybe let's use `.dvc/tmp/index` instead?